metaworld/policies/mujoco/sawyer_xyz/base.py

```diff
@@ -0,0 +1,67 @@
+import abc
+import warnings
+
+import numpy as np
+
+from metaworld.policies.base import Policy
+
+
+class SawyerXYZPolicy(Policy, metaclass=abc.ABCMeta):
+    def __init__(self, pid_xyz):
+        self._pid_xyz = pid_xyz
+
+    def get_action(self, obs):
+        """Gets an action in response to an observation.
+
+        Args:
+            obs (np.ndarray): Observation which conforms to env.observation_space
+
+        Returns:
+            np.ndarray: 4 element array representing the action to take
+        """
+        xyz_curr = obs[self.obs_idx_of_effector_xyz]
+        xyz_error = self.xyz_goal(obs) - xyz_curr
+        xyz_response = self._pid_xyz.response_to(xyz_error)
+
+        if np.any(np.absolute(xyz_response) > 1.):
+            warnings.warn('PID constants may be too high. Response gets clipped at [-1, 1]')
```

Comment:
can you do all of this will only `PD` loop? The `I` in PID is stateful!

Reply:
Yes; in fact that's what is happening here. The PID class has a flag that specifies whether it should be stateful or not. If it's stateless, it won't use the `I` and `D`. The tests ensure that (1) `self._pid_xyz` is the only instance variable and (2) that `stateful==False`.

I realize that calling it `PID` even when it's just `P` could be confusing, but I wrote the class before I understood the need to keep things stateless, and I kept it in case we ever want to write stateful policies.

Reply:
Let's not keep things we don't use, especially if it could lead to accidental improper usage (footguns). Someone in the future might not understand that `ID` makes the policy stateful.

Can you change it to just be a `PController` class?